builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -509,9 +514,31 @@ func (b *backend) pathGetRawIssuer(ctx context.Context, req *logical.Request, da
 		return nil, err
 	}
 
-	certificate := []byte(issuer.Certificate)
-
 	var contentType string
+	var certificate []byte
+
+	// tmp
+	responseHeaders := map[string][]string{}
+	if hasHeader(headerIfModifiedSince, req) {
```

Comment:
If we do do that refactor, I wonder if we could put all of this in a helper? `resp, return := sendNotModifiedResponseIfNecessary(...)`